docs/content/en/blog/news/february-2022-update.md

```diff
@@ -0,0 +1,97 @@
+
+### What's changed
+---
+
+Since the last report, PipeCD team has introduced 4 releases ([v0.24.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.0), [v0.24.5](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.5), [v0.25.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.25.0), [v0.25.1](https://github.com/pipe-cd/pipecd/releases/tag/v0.25.1)). This blog post walks you through their notable changes. For all other changes, please check out each release note to see.
+
+#### Introducing Label mechanism
+
+Label concept has been introduced. Labels are key/value pairs that are attached to PipeCD resources such as application, deployment... As a developer, you can use labels to add identifying attributes to them. We believe that supporting the Label will give us more flexibility on grouping, filtering as well as managing access control on PipeCD resources.
+
+Application labels can be specified via the `labels` field in the application configuration file while deployment labels are inherited from its application.
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: KubernetesApp
+spec:
+  name: example
+  labels:
+    env: prod
+    team: product
+```
+
+From `v0.24.0`, application labels are shown on the application list and detail page. All filter forms also have a new label box to allow filtering resources by their labels.
+
+#### Environment and Deployment Configuration File were deprecated
+
+Since the Label was introduced as above, Environment was becoming a subset of the Label concept. So we decided to deprecate the Environment concept from `v0.24.0`. Although it is still being displayed on the web console, it will be removed completely from the next release. So make sure to switch to using Label instead.
+
+The deployment configuration file (`.pipe.yaml`) has been deprecated as well. Please use the application configuration file instead, it has the same format but with some new fields such as `name`, `description`, and `labels`. By using this new config file, all application information can be stored and managed in your Git repository. The new config file also gives PipeCD the ability to detect unregistered applications in Git repository to suggest users to add on the control plane.
+
+Basically, a deployment configuration file can be migrated to be an application configuration file just by adding the `name` field.
+
+```yaml
+# Old deployment configuration file
+apiVersion: pipecd.dev/v1beta1
+kind: KubernetesApp
+spec:
+  ...
+```
+
+```yaml
+# New application configuration file
+apiVersion: pipecd.dev/v1beta1
+kind: KubernetesApp
+spec:
+  name: example # Add this field.
+  labels:
+    env: prod
+    team: product
+  ...
+```
+
+If you are having many applications, you can use this [pipectl command](/docs/user-guide/command-line-tool/#migrating-deployment-configuration-files-to-application-configuration-files) to migrate a bunch of your files seamlessly.
```

Comment:
nits, I think instead of `/docs` should use `/docs-v0.x.x` to specify the version containing the migration command since this command may be deleted later and the docs can be removed.

Reply:
Nice idea.